backend/router.js

```diff
@@ -11,9 +12,68 @@ app.use(cors())
+                    to the question: ${message} Lastly, ask a follow-up question to the user based on their response. Make the question relevant to them yet still broad enough to allow them to answer on their own terms. 
+                    Format for the follow up question should be exactly as follows: "As a follow-up question, (the followup question)"`
+                    : !lastQuestion &&!doFollowUp?
+                    `You are a interviewer conducting a behavioral interview. Make sure to talk in the first person as if this was a normal conversation between two people; do not include quotes around your response.  
+                    The user is about give their response to a question that you have 
+                    asked. Please provide a relevant reaction to their answer and say what you liked about their response. Remember to keep this concise. Here is the question that was asked: ${context}. Here is the user's response 
+                    to the question: ${message}. Do not ask any type of question within your response. Say something to end this thought and mention going on to the next question. `
+                    :
+                    `
+                    You are a interviewer conducting a behavioral interview. Make sure to talk in the first person as if this was a normal conversation between two people; do not include quotes around your response.  
+                    The user is about give their response to a question that you have 
+                    asked. Please provide a relevant reaction to their answer and say what you liked about their response. Remember to keep this concise. Here is the question that was asked: ${context}. Here is the user's response 
+                    to the question: ${message}. Do not ask any type of question within your response. Say something to end this thought.
+                    Lastly, since this was the last question, you are done conducting the interview.
+                    Express your thanks for getting to interview the person and mention that there will be post interview feedback via our post interview review. 
+                    `
+                }
+            ],
+            model: "llama3-70b-8192"
+        });
+
+
+        const botResponse = response.choices[0]?.message?.content || "I didn't understand that.";
+
+
+        // Use regex (regular expression) to extract follow-up question
+        const followUpQuestionMatch = botResponse.match(/follow-up question, (.*)$/);
+        const followUpQuestion = followUpQuestionMatch ? followUpQuestionMatch[1].trim() : null;
+        
+        
+
+        res.json({
+            response: botResponse,
+            followUp: followUpQuestion
+        });
+    } catch (error) {
+        console.error("Error getting chat completion:", error);
+        res.status(500).json({ error: "Error getting chat completion" });
```

Comment:
these strings seem to be used quite a few times ("error getting chat completion" and "i didn't understand that")- can we move them to constants?

Reply:
Okay, will do. That's a good point.

Reply:
Yeah I would suggest a separate file just for constants